x-pack/plugins/security_solution/public/detections/components/alerts_table/alerts_grouping.tsx

```diff
@@ -76,8 +76,8 @@ const GroupedAlertsTableComponent: React.FC<AlertsTableComponentProps> = (props)
 
   const { onGroupChange, onGroupToggle } = useMemo(
     () => ({
-      onGroupChange: (param: { groupByField: string; tableId: string }) => {
-        telemetry.reportAlertsGroupingChanged(param);
+      onGroupChange: ({ groupByField, tableId }: { groupByField: string; tableId: string }) => {
+        telemetry.reportAlertsGroupingChanged({ groupByField, tableId });
```

Comment:
following the comment [here](https://github.com/elastic/kibana/pull/175749#issuecomment-1916654182), I updated to select the objects that are going to be sent to telemetry, I also added a unit test to cover this logic